### ISSUE TITLE
Icons that are linked are flagged by page speed insights that "Links do not have discernible names"

### ISSUE THREAD

**Issue by shsteimer** [adobe/aem-boilerplate#242]
### Expected Behaviour

When an icon is linked, attributes should be set not cause accessibility issues. I think this could be handled in a few ways (aria label/aria-labelledby, sr-only text, etc.), but ideally if the svg contains a `<title>`, that is used, and if not, it falls back to the icon name.

### Actual Behaviour

No aria label or sr text is set, so links are flagged by the accessibility checks. This seems to only be a problem for the svg use references from the sprite, when svgs are inlined, if those svgs have a `<title>` it seems to be fine from my testing.

### Reproduce Scenario (including but not limited to)

see https://pagespeed.web.dev/analysis/https-surest-com/yknkix6bfs?form_factor=mobile but reproducible across many Franklin sites who have icons that are linked (very common in header and footer)

**Comment by shsteimer** [comment]
closing, as icon decoration has changed a lot since this opened.
